Tests/MongoSwiftTests/DocumentTests.swift

```diff
@@ -583,4 +585,116 @@ final class DocumentTests: MongoSwiftTestCase {
 
         expect(noops).to(equal(["null": 5, "maxkey": "hi", "minkey": false]))
     }
+
+    private struct CollectionConformance {
```

Comment:
we already have a Document+SequenceTests.swift. How about we just create a new test class in a new file Document+CollectionTests.swift?

Reply:
Done.